Header header
uint64 counter
uint64 scale
geometry_msgs/Pose maze_origin